### ISSUE TITLE
compile error with pre-release 1.4.533

### ISSUE THREAD

**Issue by anielsen001** [mps9506/quarto-cv#10]
I'm trying to run with the pre-release version of quarto (1.4.533) and getting this error:
```
ERROR: 
compilation failed- error
LaTeX Error: Lonely \item--perhaps a missing list environment.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.240 ...m[\citeproctext]{ref-student1938comparison}
                                                  


see template.log for more information.
```

When I run with the current release version of quarto 1.3.450, everything compiles without error. I'm not actually sure what the difference is. I'm running a clean checkout of the v1.0.2 branch of this project.

here's the log file for the failed compilation.
[template.log](https://github.com/mps9506/quarto-cv/files/13794528/template.log)

**Comment by mps9506** [comment]
Thanks @anielsen001, I can confirm this (I had to roll back to the release version of quarto myself). This is linked to underlying updates in the pandoc/citeproc engines. I am trying to get a fix rolled out, but my suggestion for now is to use the v1.3.x of quarto.

**Comment by anielsen001** [comment]
Thank you for looking at this. I've been trying to use conditional spans to build my cv for different purposes. These apparently were in quarto 1.2, removed in 1.3 and coming back in 1.4. I rolled back to v1.0.0 of quarto-cv and quarto v1.2 and it worked.

**Comment by mps9506** [comment]
Please try the following with quarto 1.4.x and let me know if it gives you issues.

`quarto use template mps9506/quarto-cv@v2.0.0` or `quarto install extension mps9506/quarto-cv@v2.0.0`

It seems to work fine on my end, but is not backwards compatible with quarto 1.3.x.

**Comment by anielsen001** [comment]
thank you, I tried it out and it worked for me with the pre-release 1.4.x, no issues.

**Comment by mps9506** [comment]
Just released v2.0.0 (to install by default) that works with quarto 1.4.
